.github/workflows/build-package-docs.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 
 name: Ansible package docs build
-on:
+"on":
```

Comment:
FWIW, dropping a config like https://github.com/ansible/awx-plugins/blob/e5f7468/.yamllint into the repo would allow these.

Reply:
Generally enabling `on` just for a handful of special YAML files sounds like a bad idea.

Reply:
Oh, there's an issue about this on the tracker.
Apparently, there's an option to allow this just in YAML keys: https://github.com/adrienverge/yamllint/issues/559#issuecomment-1674327721. I'll probably go update my typical config...